cmd/reset.go

```diff
@@ -13,7 +13,8 @@ var (
 		Long:  "Rebuilds the hooks as defined in the .hooks.yaml file.",
 		Run: func(cmd *cobra.Command, args []string) {
 			hookzHeader()
-			fmt.Println("Resetting git hooks...")
+			fmt.Println("Resetting git hooks\n")
```

Comment:
```suggestion
			fmt.Println("Resetting git hooks")
			fmt.Println()
```
This seems to be causing a test failure.